lambdas/db-migration/src/migrations/20201014105058_create_granules_table.ts

```diff
@@ -24,55 +24,55 @@ export const up = async (knex: Knex): Promise<void> =>
       .float('duration')
       .comment('Ingest duration');
     table
-      .float('timeToArchive')
+      .float('time_to_archive')
       .comment('Number of seconds granule took to archive');
     table
-      .float('timeToProcess')
+      .float('time_to_process')
       .comment('Number seconds granule took to complete "processing"');
     table
-      .integer('productVolume');
+      .integer('product_volume');
     table
       .jsonb('error')
       .comment('JSON error object');
     table
-      .text('cmrLink')
+      .text('cmr_link')
       .comment('Link to granule in the CMR API');
     table
       .text('execution')
       .comment('Step Function Execution link')
       .notNullable();
     table
-      .text('granuleId')
+      .text('granule_id')
       .comment('Granule ID')
       .notNullable();
     table
-      .text('pdrName')
+      .text('pdr_name')
       .comment('PDR associated with the granule');
     table
       .text('provider')
       .comment('Provider granule is associated with');
     table
-      .timestamp('beginningDateTime')
+      .timestamp('beginning_date_time')
       .comment('Date granule started');
     table
-      .timestamp('endingDateTime')
+      .timestamp('ending_date_time')
       .comment('Date granule completed');
     table
-      .timestamp('lastUpdateDateTime')
+      .timestamp('last_update_date_time')
       .comment('Timestap for last update');
     table
-      .timestamp('processingEndDateTime')
+      .timestamp('processing_end_datetime')
```

Comment:
should this be `processing_end_date_time`?

Reply:
Agree with these suggestions, otherwise this is good to go